server/server_test.go

```diff
@@ -228,18 +228,10 @@ func TestAcceptEncoding(t *testing.T) {
 	defer leaktest.AfterTest(t)
 	s := StartTestServer(t)
 	defer s.Stop()
-	// We can't use the standard test client. Create our own.
```

Comment:
odd. apparently Proxy and DisableCompression have no impact on this test. I thought that was the whole point of the test. Oh well, go ahead and remove it then.